src/renderer/components/+landing-page/landing-page.tsx

```diff
@@ -1,40 +1,23 @@
 import "./landing-page.scss";
 import React from "react";
-import { observable } from "mobx";
 import { observer } from "mobx-react";
-import { clusterStore } from "../../../common/cluster-store";
-import { workspaceStore } from "../../../common/workspace-store";
-
+import { Workspace, workspaceStore } from "../../../common/workspace-store";
+import { WorkspaceOverview } from "./workspace-overview";
+import { PageLayout } from "../layout/page-layout";
 @observer
 export class LandingPage extends React.Component {
```

Comment:
Will change the naming (easier to track differences here by not changing the name/filename yet)